pages/linux/lid.md

```diff
@@ -0,0 +1,9 @@
+# lid
+
+> Query ID database and report results.
+> On Fedora and Arch Linux, the binary name `lid` is taken by another program. See `tldr libuser-lid`.
+> More information: <https://example.org/>.
```

Comment:
Also, I couldn't find `id-utils`'s `lid` on [manned.org](https://manned.org/browse/search?q=lid). [die.net](https://linux.die.net/man/1/lid)'s page is out of date, and I really do not want to link [unix.com](https://www.unix.com/man-page/debian/1/LID/).

Reply:
```suggestion
> More information: <https://www.gnu.org/software/idutils/>.
```
Found the GNU version here: https://www.gnu.org/software/idutils/

This link, given it's the homepage thereof, would be preferable over a man page.

It also has a link to a full manual.

Reply:
Ah thanks for the reminder. I must have missed the notification on @sbrl's reply. Updated.